IDE/MonoDevelop/MonoDevelop.MonoGame/templates/MonoGameProject.xpt.xml

```diff
@@ -28,6 +28,7 @@
 			<Files>
 				<File name="Game1.cs" src="Common/Game1.cs" />
 				<File name="Program.cs" src="Common/Program.cs" />
+				<File name="app.manifest" src="Common/app.manifest" />
```

Comment:
@mrhelmut asked for that. Not sure why exactly, but he looked into this more than I did.